### DISCUSSION TITLE
Alpine vs Debian

### DISCUSSION THREAD

**Discussion by gregewing** [nicolargo/glances#2237] [General]
This is really an observation:

The docker images which claim to be based on Debian ( currently as of today ) are actually running Alpine.

based on the following document, I have downloaded and run each image listed in the section titles **Docker: the fun way**
https://github.com/nicolargo/glances/blob/v3.3.0/docker-files/debian.Dockerfile

- alpine-latest-full for a full Alpine Glances image (latest release) with all dependencies - (is Alpine)
- latest-full for a full **Debian** Glances image (latest release) with all dependencies **- (is Alpine)**
- alpine-latest for a basic Alpine Glances (latest release) version with minimal dependencies  - (is Alpine)
- latest for a basic **Debian** Glances image (latest release) with minimal dependencies **- (is Alpine)**
- alpine-dev for a basic Alpine Glances image (development branch) with all dependencies - (is Alpine)
- dev for a basic **Debian** Glances image (development branch) with all dependencies **- (is Alpine)**

Am I looking at an old set of documentation or something ?

**Comment by RazCrimson** [comment]
We switched to Alpine pretty recently.

I think the documentation was not updated.
